packages/ui-react/src/components/TextField/TextField.tsx

```diff
@@ -69,7 +69,7 @@ const TextField: React.FC<Props> = ({
     const inputType = 'type' in otherInputProps ? otherInputProps.type : 'text';
 
     const ariaAttributes = {
-      'aria-invalid': Boolean(error && !value),
+      'aria-invalid': Boolean(error && !!value),
```

Comment:
Looks like this change led to some snapshot updates where the default error state is set to `true` now. Is it expected?

Reply:
Yes, that is to be expected! The `DemoConfigDialog` and `PersonalDetailsForm` snapshot files are updated in the instances where errors are intentionally being passed. As a result, `aria-invalid` is now correctly set to `true` for these affected fields, where they previously incorrectly were set to `false`.